pkg/policy/rule_consumers.go

```diff
@@ -112,7 +112,7 @@ func (a *AllowRule) Allows(ctx *SearchContext) ConsumableDecision {
 // RuleConsumers allows the following consumers.
 type RuleConsumers struct {
 	Coverage []*labels.Label `json:"coverage,omitempty"`
-	Allow    []AllowRule     `json:"allow"`
+	Allow    []*AllowRule    `json:"allow"`
```

Comment:
`AllowRule` contains a `Label` which contains a mutex. `go vet` complains that the mutex pointer is being copied if the `AllowRule` is not a pointer.